src/fides/api/email_templates/templates/user_invite.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Welcome to Fides</title>
+  </head>
+  <body>
+    <main>
+      <p>You've been invited to join Fides, click <a href={{admin_ui_url}}/login?invite_code={{invite_code}}&username={{username}}>here</a> to accept the invite and setup your account.</p>
```

Comment:
Could be useful to have org name here too, in addition to copy that says something about how this link will expire in 24 hours.